osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -517,38 +516,64 @@ public InfoLabel(BeatmapStatistic statistic)
 
             private class DifficultyColourBar : Container
             {
-                private readonly StarDifficulty difficulty;
+                [Resolved]
```

Comment:
This should also be reverted. It has issues and I'd rather address them in a separate PR.